main.tf

```diff
@@ -0,0 +1,59 @@
+# Repository configuration module
+resource "github_repository" "coremaker_repo" {
```

Comment:
let's use a more common and meaningful name for all resources.

Reply:
Done